.clinerules/workflows/build.md

```
@@ -0,0 +1,63 @@
+```powershell
+dotnet clean QuanTAlib.sln --configuration Debug --verbosity minimal
+```
+**Expected**: Clean completed without errors.
+
+### 3. Build Solution
+```powershell
+dotnet build QuanTAlib.sln --configuration Debug --no-restore
+```
+**Expected**: Build succeeded with 0 errors. Note any warnings for fixing.
+
+### 4. Build Solution
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)